### PR TITLE
Create basic-registry.adoc

### PR DESCRIPTION
**Pull request by dmuelle**

### CODE REVIEW

File: ref/general/basic-registry.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+== The Application Security feature
+
+The `appSecurity` feature provides the core capability for the Java EE Security API. This feature must be enabled in the `server.xml` file in order to configure a basic user registry. For more information, see link:/docs/ref/feature/#appSecurity-3.0.html[the Application Security feature].
+
+== Basic user registry for test environments
+
+Instead of being hosted externally, Open Liberty's basic user registry is configured directly in the `server.xml` file, or in an adjacent xml file on the server. Although this configuration is not secure enough for production environments, it can help simplify authentication and authorization scenarios during development and testing. The basic user registry provides a fast and easy way to test the access to your protected application.
```
Comment (Charlotte-Holt) [inline ref/general/basic-registry.adoc]: Should `xml` in "...or in an adjacent xml file on the server." be all caps (`XML`)?

File: ref/general/basic-registry.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+     <user name="Hiro" password="hiropwd" />
+     <group name="admin">
+       <member name="Asha" />
+     </group>
+</basicRegistry>
+----
+
+== QuickStart security with Open Liberty
+
+Another option for configuring a basic user registry for development environments is to add the `quickStartSecurity` element to the server.xml file, as shown in the following example:
```
Comment (Charlotte-Holt) [inline ref/general/basic-registry.adoc]: Add back ticks around server.xml file (`server.xml` file

File: ref/general/basic-registry.adoc (hunk 1 above)
Comment (dmuelle) [reply ref/general/basic-registry.adoc]: yes- thanks good catch

**Comment by dmuelle** [comment]
thanks for the review @Charlotte-Holt - comments addressed

**Pull request merged**